### ISSUE TITLE
php 8.0.1 and 7.4.14 (and 7.3.26)

### ISSUE THREAD

**Issue by webmaster777** [oerdnj/deb.sury.org#1516]
**Frequently asked questions**

- [x] I have read [Frequenty Asked Questions](https://github.com/oerdnj/deb.sury.org/wiki/Frequently-Asked-Questions)

**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Builds of php 8.0.1 and 7.4.14

**Describe alternatives you've considered**
Waiting. I don't know how or whether you get notified of new releases. If you are, consider this a friendly nudge. I know your time is sparse. 

Please understand that this issue tracker is specifically meant to report issues with the packages and with the repositories. I won't teach you the basics of system administration, I won't help you with bugs in the software itself. For both, there are better sites where are people with more free time that allows them to be more friendly to newcomers.

Please make sure you read [Dear anonymous internet user asking for help...](https://berthub.eu/articles/posts/anonymous-help/) by Bert Hubert and [How to Report Bugs Effectively](https://www.chiark.greenend.org.uk/~sgtatham/bugs.html) by Simon Tatham.

If you understand the purpose of this issue tracker and describe your **request** accurately (where the template below will help), I might be able to help you.

**Distribution (please complete the following information):**
 - OS: Ubuntu 18/20
 - Architecture: amd64
 - Repository: Ubuntu PPA

**Package(s) (please complete the following information):**
```
php8.0:
  Installed: (none)
  Candidate: 8.0.0-1+ubuntu18.04.1+deb.sury.org+1
  Version table:
     8.0.0-1+ubuntu18.04.1+deb.sury.org+1 500
        500 http://ppa.launchpad.net/ondrej/php/ubuntu bionic/main amd64 Packages
php7.4:
  Installed: 7.4.11-6+ubuntu18.04.1+deb.sury.org+1
  Candidate: 7.4.13-1+ubuntu18.04.1+deb.sury.org+1
  Version table:
     7.4.13-1+ubuntu18.04.1+deb.sury.org+1 500
        500 http://ppa.launchpad.net/ondrej/php/ubuntu bionic/main amd64 Packages
 *** 7.4.11-6+ubuntu18.04.1+deb.sury.org+1 100
        100 /var/lib/dpkg/status
```

this server uses 7.4, but 8.0 is used on our dev environments.

**Additional context**
Love your work and effort ❤️

**Comment by oerdnj** [comment]
I've uploaded the new versions this morning, so they are still building or already waiting for you to update ;)

**Comment by webmaster777** [comment]
re: PPA

I see 5.6-7.2 have been published, but 7.3-8.0 have no updates yet, and there are no builds pending on launchpad. Did something get stuck?

![image](https://user-images.githubusercontent.com/2650744/104421572-ccc91280-557b-11eb-82aa-5be6e481cf18.png)

(filtered for bionic)

**Comment by oerdnj** [comment]
Thanks for the poke, all of them should be uploaded now. For some reason, I skipped the real release and did just the patched ones.
